osquery/extensions/interface.cpp

```diff
@@ -51,22 +51,19 @@ void ExtensionHandler::call(ExtensionResponse& _return,
   }
 
   PluginResponse response;
-  PluginRequest plugin_request;
-  for (const auto& request_item : request) {
-    // Create a PluginRequest from an ExtensionPluginRequest.
-    plugin_request[request_item.first] = request_item.second;
-  }
+  PluginRequest plugin_request = request;
+  // for (const auto& request_item : request) {
```

Comment:
Ah, sorry for confusion, I was referring to these Xp

Reply:
oh that's an unrelated change from another branch I was experimenting with.
I'll remove it.